Makefile

```diff
@@ -119,15 +119,18 @@ manifests: controller-gen ## Generate manifests e.g. CRD, RBAC etc.
 e2e-telepresence: cleanup-e2e install ## Run e2e tests locally using go build while also setting up a proxy e.g. make e2e-telepresence test=replica_set cleanup=true
 	telepresence connect; \
 	eval $$(scripts/dev/get_e2e_env_vars.py $(cleanup)); \
-	go test -v -timeout=30m -failfast ./test/e2e/$(test); \
+	go test -v -timeout=30m -failfast $(options) ./test/e2e/$(test) ; \
 	telepresence quit
 
-e2e-k8s: cleanup-e2e install e2e-image ## Run e2e test by deploying test image in kubernetes.
-	python scripts/dev/e2e.py --perform-cleanup --test $(test)
+e2e-k8s: cleanup-e2e install e2e-image ## Run e2e test by deploying test image in kubernetes without cleanup of the resources.
```

Comment:
grepping shows that we don't use this target in our workflow anywhere, making this change most likely safe.

Reply:
Can we make this target take additional arguments instead? (since `e2e.py` supports additional flags which we might want to use)

Reply:
added in: 41ec97481ed2b31e3d59f1105994cd902a45b855